index.html

```diff
@@ -0,0 +1,169 @@
+          </svg>
+          </li>
+        <li>Spanish</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </nav>
+    <!--Social media nav bar -->
+    <!--Big screen nav bar -->
+    <nav class="desktop__nav">
+      
+      <a href="./index.html"><img src="./images/big_logo.png"></a>
+      
+      <ul class="desktop__options">
+        <li>
+          <a href="./about.html">About</a>
+        </li>
+        <li>
+          <a href="#program">Program</a>
+        </li>
+        <li>
+          <a href="#speakers">Speakers</a>
+        </li>
+        <li>
+          <a href="#partner">Partner</a>
+        </li>
+        <li class="campaign">
+          Campaign
+        </li>
+      </ul>
+    </nav>
+    <!--Big screen nav bar-->
+    <nav class="nav__bar">
+      <img class="nav__menu" src="./images/mobile_menu 1.png" alt="Menu">
+       
+    </nav>
+
+    <!-- Hamburguer menu elements-->
+  <section class="hamb__menu" hidden>
+    <div>
+      <a href="./about.html">
+        <h3 class="hamb__h">About</h3>
+      </a>
+      <a href="#program">
+        <h3 class="hamb__h">Program</h3>
+      </a>
+      <a href=#speakers>
+        <h3 class="hamb__h">Speakers</h3>
+      </a>
+    </div>
+    <svg class="hamb__close" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"
+        fill="#979493" />
+    </svg>
+  </section>
+  <!-- Hamburguer menu elements-->
+
+
+
+
+    <section class="first__section">
+      <div class="rocket__global">
+        <h3>"Hello Rocket world!"</h3>
+        <h1>ROCKET GLOBAL SUMMIT 2023</h1>
```

Comment:
- [ ] I noticed that you use the `<h1>` tag many times on the same page. I would like to recommend you use only one single `<h1>` tag for each page (index and about)  because using only one H1 creates hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about. Please fix it :+1: